addons/breathing/functions/fnc_getETCo2.sqf

```diff
@@ -0,0 +1,61 @@
+#include "..\script_component.hpp"
+/*
+ * Author: apo_tle
+ * Calculate patient's ETCo2 level based on status of their vitals. ETCo2 is not its own vital as it does not need to be,
```

Comment:
delete unnecessary description, "Calculate patient's ETCo2 level based on status of their vitals." is enough